src/types/hapticEffects.ts

```diff
@@ -0,0 +1,4 @@
+export type HapticEffects = {
```

Comment:
I'd suggest we don't hardcode the haptic motor locations since it could change in future hardware versions. Also, the number of motors could change. Instead, we could have a type per device model.